guides/plugins/plugins/api/custom-pricing.md

```diff
@@ -0,0 +1,97 @@
+# Custom Pricing
```

Comment:
Please add a corresponding entry in the `SUMMARY.md` file for both new pages in the PR.